### ISSUE TITLE
Editing always fails

### ISSUE THREAD

**Issue by augomat** [mbdavid/LiteDB.Studio#46]
Compiled last version (1cda397), editing of attached db always gives me "Current document not found"

![image](https://user-images.githubusercontent.com/24647634/103370323-cee79780-4acc-11eb-8119-283557155ee8.png)

![image](https://user-images.githubusercontent.com/24647634/103370346-ddce4a00-4acc-11eb-9814-3fe253860c84.png)

Something seems to be odd with the tags, not sure how this is supposed to work with the bindings but tags are always null.

![image](https://user-images.githubusercontent.com/24647634/103390246-b77fcd00-4b13-11eb-8607-68dbbd05b750.png)

**Comment by markusstephany** [comment]
It seems that the UIExtensions class broke the grid as cell tags are not set anymore to document data. Also the overall handling has changed so I think the GrdResult_CellBeginEdit in MainForm does not work anymore. I tried to fix those issues (and it seems to work...) in my fork (markusstephany/LiteDB.Studio, branch "spy" and maybe master). but as I faced that issue after quite a few other modifications, my fork does also contain other changes (e.g. overall refactoring, settings dialog), so it will take some time to find just those changes related to this bug...

Br Markus

**Comment by augomat** [comment]
Thanks al lot, will try it out!
It's a pity that much of the improvements/fixes appear to happen in forks which are never merged back.
However, as you pointed out, the diff of yours is huge, mainly due to code formatting as far as I have seen. That's unfortunate.

**Comment by markusstephany** [comment]
Yes, most of the changes are cosmetics (personal ReSharper code style applied). Except for the settings dialog and the editing issues. Maybe someone can back-port the latter mods to the original code base...

**Comment by Paul-Williams** [comment]
This is still broken! Makes the studio app next to useless.

**Comment by Paul-Williams** [comment]
Latest version of [this](https://github.com/markusstephany/LiteDB.Studio) fork does not solve the problem.

Steps to reproduce:

1. Open DB
2. Run 'collection select all' query (SELECT $ FROM MyTable)
3. Edit text 'field' in query results (NB: Editing automatically clears value from cell in grid - another bug?)
4. Navidate away from edited row
5. Error: Current document was not found. Try run your query again,
